.kitchen.yml

```diff
@@ -17,6 +17,7 @@ platforms:
 - name: macosx-10.10
   driver:
     box: chef/macosx-10.10
+- name: win81x64-enterprise
```

Comment:
what is this box? where does it come from? I presume it's not publicly available?